vms/shift/models.py

```diff
@@ -23,32 +24,10 @@ class Shift(models.Model):
         ],
         blank=True,
         null=True,
-    )
-    city = models.CharField(
-        max_length=75,
-        validators=[
-            RegexValidator(r'^[(A-Z)|(a-z)|(\s)|(\-)|(\')]+$', ),
-        ],
-        blank=True,
-        null=True,
-    )
-    state = models.CharField(
-        max_length=50,
-        validators=[
-            RegexValidator(r'^[(A-Z)|(a-z)|(\s)|(\-)]+$', ),
-        ],
-        blank=True,
-        null=True,
-    )
-    country = models.CharField(
-        max_length=75,
-        validators=[
-            RegexValidator(r'^[(A-Z)|(a-z)|(\s)|(\-)|(\')]+$', ),
-        ],
-        blank=True,
-        null=True,
-    )
-
+    )
    
```

Comment:
Please correct this ^M thing. It has something to deal with the line ending character.